src/patternfly/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -161,15 +161,15 @@ import './MenuToggle.css'
 ### Plain
 ```hbs
 {{#> menu-toggle menu-toggle--IsPlain=true menu-toggle--attribute='aria-label="Actions"'}}
```

Comment:
I haven't tried it, but the hbs looks like it might be wired up so you can write it this way?

```suggestion
{{> menu-toggle menu-toggle--IsPlain=true menu-toggle--attribute='aria-label="Actions"' menu-toggle--icon="ellipsis-v"}}
```

Reply:
@mcoker the additional changes you made for this in the examples/demos all look good.